device/dhcp/servers/v4/dhcpv4server.yaml

```diff
@@ -0,0 +1,30 @@
+components:
+  schemas:
+    DhcpServer.V4:
+      description: >-
+        Configuration for emulated DHCPv4 Server.
+      type: object
+      x-status:
+        status: under_review
+      required: [name, ipv4_name, address_pools]
+      properties:
+        name:
+          x-include: ../../../common/common.yaml#/components/schemas/Named.Object/properties/name
+          x-field-uid: 1
+        ipv4_name:
```

Comment:
According to current model `gateway` within `ipv4_addresses` is a required field.  I have checked some sample DHCP server config in IxNetwork. It looks like Gateway of IPv4 interface (where DHCP server place top of it) not configured (set as `0.0.0.0`).
Then have these options:
- User have to explicitly set `0.0.0.0` as gateway.
- Otherwise remove `required` tag from `gateway` which is probably not a good option.